lib/agent.js

```diff
@@ -366,6 +376,24 @@ Agent.prototype.forceHarvestAll = function forceHarvestAll(callback) {
     promises.push(errorEventPromise)
   }
 
+  // TODO: There are 3 diff config flags
+  // application_logging.enabled overrides the others
+  // application_logging.forwarding.enabled - enables log forwarding
+  // application_logging.metrics.enable toggles collecting metrics
```

Comment:
Leftover TODO here that isn't quite clear what else needs to be done. Might recommend rewording to the action needed if the TODO is still warranted.

Reply:
this was just for me but i added a comment explaining the order of precedence